raster/r.info/main.c

```diff
@@ -115,8 +115,9 @@ int main(int argc, char **argv)
 
     if (hflag->answer &&
         (gflag->answer || rflag->answer || sflag->answer || eflag->answer))
-        G_fatal_error(_("Flags -%c and -%c/%c/%c are mutually exclusive"),
-                      hflag->key, gflag->key, rflag->key, eflag->key);
+        G_fatal_error(_("Flags -%c and -%c/%c/%c/%c are mutually exclusive"),
+                      hflag->key, gflag->key, rflag->key, eflag->key,
+                      sflag->answer);
```

Comment:
Noticed that the placeholders can't be reordered in translations. They weren't before your PR too